### PR TITLE
Toasts v1

### PR DESCRIPTION
**Pull request by SaraVieira**
## Before requesting a review

- [ ] if the PR is WIP - use draft mode
- [x] made a self-review
- [x] added inline comments where things may be not obvious (the "why", not "what")
- [ ] what kind of review is needed?
  - [x] conceptual
  - [ ] detailed
  - [x] with testing

## Test cases

- [ ] step by step interaction description and what is expected to happen

## Before merging

- [x] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio-designer/blob/main/apps/designer/docs/test-cases.md) document
- [ ] added tests
- [x] if any new env variables are added, added them to `.env.example` and the `designer/env-check.js` if mandatory

image
image

closes https://github.com/webstudio-is/webstudio-designer/issues/165

### CODE REVIEW

File: packages/design-system/src/components/toast.tsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+  margin: 0,
+  listStyle: "none",
+  zIndex: "$max",
+  outline: "none",
+});
+
+const StyledToast = styled(ToastPrimitive.Root, {
+  borderRadius: 6,
+  boxShadow:
+    "hsl(206 22% 7% / 35%) 0 $space$2 $space$6 -$space$2, hsl(206 22% 7% / 20%) 0 $space$2 $space$4 -$space$3",
```
Comment (SaraVieira) [inline packages/design-system/src/components/toast.tsx:44]: Is there a way to add our colors with transparency?

Comment (kof) [reply packages/design-system/src/components/toast.tsx:44]: hmm not sure how to deal with it, we can  define colors in hsl and then have a function that changes opacity parameter, but we need to discuss with @taylornowotny when would you use that and why

Comment (SaraVieira) [reply packages/design-system/src/components/toast.tsx:44]: I was thinking RGBA, but that's an implementation detail.

For shadows and stuff we should definitely have something that allows to pass transparency

Comment (kof) [reply packages/design-system/src/components/toast.tsx:44]: the question is when do you need to pass a non-static, predefined transparency

Comment (SaraVieira) [reply packages/design-system/src/components/toast.tsx:44]: I do agree, at remote we had these predefined so for example $highContrast_20 would be 20% opacity

File: packages/design-system/src/components/toast.tsx [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+          </Box>
+          <StyledTitle>{toast.message}</StyledTitle>
+        </StyledToast>
+      ))}
+      <StyledViewport />
+    </ToastPrimitive.ToastProvider>
+  );
+};
+
+export const toast = BaseToast;
```
Comment (kof) [inline packages/design-system/src/components/toast.tsx:115]: the mapping is weird, if its just a function, not a constructor and we are calling it as such, why not import it as `toast as well`
```suggestion
export {toast}
```

File: packages/design-system/src/index.ts [hunk 3]
```diff
@@ -11,6 +11,7 @@ export {
   theme,
   type CSS,
 } from "./stitches.config";
+export { toast, Toaster } from "./components/toast";
```
Comment (kof) [inline packages/design-system/src/index.ts:14]: I started exporting specific components but its tedious, would only do that if there is something to hide I a component

```suggestion
export * from "./components/toast";
```

**Review by kof — APPROVED** [review]
a few nits, looking good, lets merge it, then wait for @taylornowotny's suggestion for design and update, need some todo or a new issue for that, a new issue assigned to Taylor is better

File: packages/design-system/src/components/toast.tsx (hunk 2 above)
Comment (SaraVieira) [reply packages/design-system/src/components/toast.tsx]: Done!

File: packages/design-system/src/index.ts (hunk 3 above)
Comment (SaraVieira) [reply packages/design-system/src/index.ts]: Also done!

**Pull request merged**